.github/workflows/linting.yml

```
@@ -46,6 +46,16 @@ jobs:
       - name: Check License Headers
         uses: nilp0inter/search-legal-headers@v0.1.0
 
+  licenses:
+    name: Dependency licenses
```

```suggestion
    name: Dependency Licenses
```

Nit: We use caps for these names in most places.

---

Done (across all PRs), thanks!